src/components/ArticleId.jsx

```diff
@@ -12,11 +12,12 @@ function ArticleId() {
     const [comments, setComments] = useState([])
     const [isLoading, setIsLoading] = useState(false)
     const [isCommentsLoading, setIsCommentsLoading] = useState(false)
+    const [vote, setVote] = useState(false)
```

Comment:
Anything to do with voting isn't required in this ticket, so really we shouldn't have this, at this moment in time.